src/app/oxford-debate/setup/page.tsx

```diff
@@ -37,21 +92,20 @@ export default function OxfordDebateSetup() {
       <div className="max-w-2xl mx-auto mt-8 flex flex-col gap-6 lg:gap-4 px-4">
         <DebateConfStringsPanel />
         <hr className="border-b-2 rounded border-neutral-800 my-2" />
+        <input
+          type="file"
+          id="branding"
+          name="branding"
+          accept=".jpg, .jpeg, .png"
+          onChange={handleImageChange}
+        />
```

Comment:
This file input does not communicate to the user if some file is already uploaded. It would be nice to have a custom branding display preview of some sort to remedy this. I don't think we should focus on doing this now though, with the wizard redesign coming soon either way.